content/docs/guides/self-hosted/console.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Console
```

Comment:
Maybe a more complete, SEO-friendly page title - e.g. _"Self-Hosted Pulumi Service: Console"_ ?

Reply:
I actually had a similar title to what you suggested previously, but changed it to this because of the breadcrumbs that is shown on the current page. It would look like this (screenshot below). Not sure if there is a way to customize what gets shown in the breadcrumbs.

![Screenshot_2019-10-29 Self-Hosted Pulumi Service - Console Service](https://user-images.githubusercontent.com/1466314/67805237-d71df000-fa4d-11e9-9c98-ff5951aacdf3.png)

Reply:
I changed the title to just Pulumi Console with a recent commit due to how "Self-Hosted...." will look in the breadcrumbs.